### ISSUE TITLE
python 3 compatibility

### ISSUE THREAD

**Issue by nhoffman** [nhoffman/bioscons#22]

**Comment by nhoffman** [comment]
re 

> I forgot to take into account the mkvenv.sh script, which needs to behave slightly differently with python3, so I'll see what I can do about that as well.

The mkvenv.sh is mostly not necessary any more - we can use the system virtualenv, scons can be installed via pip. I think that a few lines in the readme should suffice. I'd start with something like this:

```
virtualenv bioscons-env
source bioscons-env/bin/activate
pip install -U pip wheel  # wheel required to pip install scons
pip install scons
```

**Comment by nhoffman** [comment]
And with respect to 

> Needs a little work to handle requirements differences since scons only supports python 3.5+ starting with version 3.0.0.

Is this true? scons 3.0 seems fine with python 2.7:

```
(scons-env) bioscons % python --version
Python 2.7.13
(scons-env) bioscons % scons --version
SCons by Steven Knight et al.:
	script: v3.0.0.rel_3.0.0:4395:8972f6a2f699[MODIFIED], 2017/09/18 13:28:16, by bdbaddog on ubuntu-16
	engine: v3.0.0.rel_3.0.0:4395:8972f6a2f699[MODIFIED], 2017/09/18 13:28:16, by bdbaddog on ubuntu-16
	engine path: ['/Users/nhoffman/src/bioscons/scons-env/lib/python2.7/site-packages/scons-3.0.0/SCons']
Copyright (c) 2001 - 2017 The SCons Foundation
```

**Comment by nhoffman** [comment]
implemented in version 0.9.0
